code/_onclick/hud/parallax.dm

```diff
@@ -20,9 +20,9 @@
 		C.parallax_fixer = new
 	for(var/obj/O in C.parallax_layers)
 		C.screen |= O
-		C.screen |= C.parallax_pmaster
-		C.screen |= C.parallax_space_whitifier
-		C.screen |= C.parallax_fixer
+	C.screen |= C.parallax_pmaster
+	C.screen |= C.parallax_space_whitifier
+	C.screen |= C.parallax_fixer
```

Comment:
again, change this so that you generate a list of things to add to the screen, then add them to the screen.

Each `C.screen` operation has an overhead because it's a special list, generate the entire list of shit to add, and add it in one go.

````
	var/list/PL = list()
	for(var/obj/O in C.parallax_layers)
		PL |= O
	C.screen |= (PL | C.parallax_pmaster | C.parallax_space_whitifier | C.parallax_fixer)
````